source/transfer_list.rst

```diff
@@ -657,5 +660,116 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+.. _optee_pagable_part_entry:
+
+OPTEE pagable part entry layout (XFERLIST_OPTEE_PAGABLE_PART)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This entry type holds the address of OPTEE pagable part.
+This address is used when OPTEED is the Secure Payload Dispatcher.
+
+.. _tab_optee_pagable_part:
+.. list-table:: OPTEE pagable part type layout
```

Comment:
I am not sure what these terms are. Please can you define them fully?

Reply:
Added more details explaination.